tests/repository_test.py

```diff
@@ -380,20 +380,58 @@ def test_swift_hook(tempdir_factory, store):
     )
 
 
-def test_golang_hook(tempdir_factory, store):
+@pytest.fixture
+def _golang_lru_cache_clear():
+    golang.get_default_version.cache_clear()
+    yield
+    golang.get_default_version.cache_clear()
```

Comment:
this shouldn't be needed

Reply:
I have added two tests, where I test default version resolving to system and to fetching latest version, so I needed to clear the `get_default_version` cache. Probably might be a stretch since we're already testing `get_default_version`, let me know if you think I should get rid of the tests.

Reply:
you can set the language version directly for these tests -- you can pass in both `default` and `system`

Reply:
done!